test/src/node.rs

```diff
@@ -713,7 +713,7 @@ impl Node {
                 let err = std::io::Error::last_os_error();
                 error!("GenerateConsoleCtrlEvent failed: {}", err);
             } else {
-                info!("GenerateConsoleCtrlEvent success");
+                ckb_logger::info!("GenerateConsoleCtrlEvent success");
```

Comment:
we have imported `use ckb_logger::{debug, error};` already, suggest import `info` also
```suggestion
                info!("GenerateConsoleCtrlEvent success");
```